notation.go

```diff
@@ -10,6 +10,9 @@ import (
 	"github.com/opencontainers/go-digest"
 )
 
+// MediaTypePayload describes the media type of the descriptor.
+const MediaTypePayload = "application/vnd.cncf.notary.payload.v1+json"
```

Comment:
This is the media type of a notary defined "wrapped" payload that currently contains only the descriptor.